widgets/contact/styles/default.less

```diff
@@ -56,12 +56,42 @@
 		}
 	}
 
+	@field_font_family: default;
+	@field_font_weight: default;
+	@field_font_size: default;
+	@field_font_color: default;
+
+	@field_background: default;
+
+	@field_margin: 0;
+	@field_padding: 0;
+
+	@field_border_radius: 0;
+	@field_border_color: default;
+	@field_border_width: default;
+	@field_border_style: default;
+
 	textarea,
 	input.sow-text-field {
-		.box-sizing(border-box);
 		display: block;
 		width: 100%;
-		border-radius: 0;
+	}
+	
+	textarea,
+	input.sow-text-field,
+	.so-premium-datepicker,
+	.so-premium-timepicker {
```

Comment:
I think we should be able to apply the `.sow-text-field` class to datepicker and timepicker fields too, without any problems. I prefer to avoid unintentionally applying styles to checkboxes.